### ISSUE TITLE
Can't understand the angle trained in network

### ISSUE THREAD

**Issue by sdukaka** [smallcorgi/3D-Deepbox#7]
In data_processing.py, 43-45 lines shows how to get theta than trianed, but i don't understand the equal in file. theat_train should be rotation_ray - angle_obj, could you explain your method?

**Comment by smallcorgi** [comment]
Hi @sdukaka,

It just orientates +pi/2, and it will subtract pi/2 in the final output.

**Comment by sdukaka** [comment]
But this angel is only the degree between camera and object center and what we want to use in training stage is the difference between theta_ray - rotation_y(the obj's own rotation)

**Comment by smallcorgi** [comment]
Yes, the outputs of network are  3D dimensions and observation angle of the object.

**Comment by jkstyle2** [comment]
@sdukaka Hi, have you figured out what the angle exactly means? I can't understand it at all. Could you please help me understand what we should train for?

**Comment by sdukaka** [comment]
I think the angle what we should train for is between camera and object
center.

jaykay <notifications@github.com> 于2019年1月23日周三 下午4:46写道：
